### PR TITLE
Pydata 0.15

### LINKED ISSUE #223: sphinx-hoverxref tooltips are not legible on the dark theme
### Describe the bug

image

### To Reproduce

_No response_

### Screenshots

_No response_

### System Details

_No response_

### Installation method

_No response_

### LINKED ISSUE #224: Remove our custom footer code when we can rely on pst 0.14
### Describe the feature

https://github.com/pydata/pydata-sphinx-theme/pull/1432

### Proposed solution

_No response_

### PR DESCRIPTION
**Pull request by Cadair**
This does actually change a few things and some colouring. I don't really have time to run them all down now.

fixes #224 
fixes #223

### CODE REVIEW

**Review by nabobalis — APPROVED** [review]

File: test_package/test_package/timerange.py
Comment (Cadair) [inline test_package/test_package/timerange.py:283]: This is a war crime.

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (Cadair) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:27]: These three were uniformly spaced in brightness. What motivated this change?

We should either change them all or none of them.

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (Cadair) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:1]: We seem to have regressed and not got `--pst-heading-color` set any more, making all our headings white (which I am not a fan of).

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (Cadair) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:70]: The jump to top thing seems to be because this colour is too close to the `--pst-color-secondary-text` colour.

Comment (Cadair) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:70]: Also curious why I didn't copy those colours in.

Comment (Cadair) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:70]: Also, also, curious as to why we don't have the background / text combos in the test page I copied out of PST.

File: test_package/test_package/timerange.py
Comment (nabobalis) [reply test_package/test_package/timerange.py:283]: This was fun to type. I will see you at The Hague

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (nabobalis) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:27]: The lack of contrast on the dark background. We want people to read grey text on a black background?

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (nabobalis) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:70]: Want me to copy more colors from the main theme we might have missed?

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (nabobalis) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:27]: I will undo the color changes we can go through them.

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css
Comment (nabobalis) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:7]: We not need all of these

File: pyproject.toml
Comment (nabobalis) [inline pyproject.toml:26]:
```suggestion
  "pydata-sphinx-theme~=0.15.0",
```

**Comment by Cadair** [comment]
It will probably be next week when I am back at work I get back to this.

**Comment by nabobalis** [comment]
Yolo it is

**Review by alasdairwilson — COMMENTED** [review]
Issues I had:

The sidebar border disconnects from the navbar when you scroll (does not do this on latest)

The navbar background is white on lightmode (grey on latest), makes navbar text unreadable.

The navbar bottom border is grey here, should be the theme orange, like the sidebar border.

**Comment by nabobalis** [comment]
Change https://pygments.org/styles/ on the light theme.

**Comment by wtbarnes** [comment]
The button that toggles light/dark is highlighted with a  full orange block when you mouse over it which looks a bit awkward:

image

**Comment by nabobalis** [comment]
Current state of the last merge: 

- [x] The navbar background is white on lightmode (grey on latest), makes navbar text unreadable.

- [x] Change [pygments.org/styles](https://pygments.org/styles/) on the light theme.

- [x] The button that toggles light/dark is highlighted with a full orange block when you mouse over it which looks a bit awkward

**Can't replicate this:**

- [ ] The sidebar border disconnects from the navbar when you scroll (does not do this on latest)

- [ ] The navbar bottom border is grey here, should be the theme orange, like the sidebar border.

**Comment by nabobalis** [comment]
Fixes https://github.com/sunpy/sunpy-sphinx-theme/issues/223

**Comment by nabobalis** [comment]
Fixes https://github.com/sunpy/sunpy-sphinx-theme/issues/216

pydata theme already uses the [quansight-labs.github.io/accessible-pygments](https://quansight-labs.github.io/accessible-pygments/) package and we select two themes that are from this.

**Comment by wtbarnes** [comment]
I had just a quick scroll through it, using both the light and dark themes, but it looks great to me!

**Review by wtbarnes — APPROVED** [review]

**Review by alasdairwilson — APPROVED** [review]

**Comment by Cadair** [comment]
I am not sure I am a massive fan of the new background colour in comparison to the old one?

image

**Comment by Cadair** [comment]
Also the double underline for the things which support hoverref isn't ideal

**Comment by wtbarnes** [comment]
Can you request changes to block on this? It has two approvals so in theory could be merged as is.

**Comment by nabobalis** [comment]
Or these are moved to issues and are patched after?

**Comment by wtbarnes** [comment]
Or that as well. I'm fine with either. I'd prefer the latter if these fixes are not straightforward.

**Comment by Cadair** [comment]
OK, I have addressed everything I would like to change, I have deliberately done it in small commits so that if people disagree with any of the stylistic choices we can easily undo.

**Comment by Cadair** [comment]
It seems that the breadcrumbs stuff is an upstream change not in CSS, so I am happy to just leave that.

**Review by alasdairwilson — APPROVED** [review]

**Review by nabobalis — APPROVED** [review]

**Comment by Cadair** [comment]
Thanks @nabobalis for the review and release. Thanks everyone else, sorry this took so long.

**Pull request merged**
